desk/app/groups.hoon

```diff
@@ -1302,13 +1302,24 @@
   ==
 ::  +safe-watch: safely watch a subscription path
 ::
-::  if .force is set, rewatch even if we have duplicate wire
+::  nb: this will not resubscribe if the subscription is
+::  still in the state, despite a leave card that might be in the
+::  queue.
 ::
 ++  safe-watch
   |=  [=wire =dock =path]
-  |=  [delay=? force=?]
+  |=  delay=?
+  ^+  cor
+  ?:  (~(has by wex.bowl) wire dock)  cor
+  =^  caz=(list card)  subs
+    (~(subscribe s [subs bowl]) wire dock path delay)
+  (emil caz)
```

Comment:
This can just be a `((safe-force-watch wire dock path) delay)` call I suppose.

`+safe-force-watch` is a funny name. Should probably be `+force-watch` instead.